api.yaml

```diff
@@ -2,51 +2,94 @@ openapi: 3.0.0
 servers:
   # Added by API Auto Mocking Plugin
   - description: SwaggerHub API Auto Mocking
-    url: https://virtserver.swaggerhub.com/Noel/TBG-Donations/1.0.10
+    url: https://virtserver.swaggerhub.com/Noel/TBG-Donations/1.0.12
   - description: "MatchBot staging"
     url: https://matchbot-staging.thebiggivetest.org.uk/v1
+  - description: "Salesforce Full (staging)"
+    url: https://thebiggive--full.sandbox.my.salesforce-sites.com/donations/services/apexrest/v2.0
 info:
-  description: The Big Give Donations public API
-  version: 1.0.10
+  description: |
+    The Big Give Donations APIs.
+
+    This contract aims to cover all endpoints except
+    `/ping` and integration webhooks. Most webhooks' structures are defined by Stripe
+    with us choosing just the endpoint.
+
+    As Salesforce & MatchBot share a Donation schema, endpoints for both are documented here
+    together; however operation `salesforceUpsert` is implemented only in Salesforce
+    and all others are implemented only in MatchBot.
+  version: 1.0.12
   title: Donations
   contact:
     email: tech@thebiggive.org.uk
   license:
     name: Apache 2.0
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
 paths:
-  /donations:
+  /people/{personId}/donations:
+    parameters:
+      - name: personId
+        in: path
+        schema:
+          type: string
+          format: uuid
+        description: Identifier of the donor
+        required: true
+
     post:
-      summary: Creates a donation
-      operationId: addDonation
-      description: Initiates a donation with pending status
```

Comment:
is there a reason for deleting the mention of the pending status? I don't know if we need to mention the status by name but may be helpful to mention that this is legitimately used when someone is only *considering* donating. "Create a donation" could be read as much more final than that. But maybe it doesn't matter much for the audience of this document.

Reply:
Not a specific one, no. This diff is actually the result of deleting the (non-`personId`) endpoint entirely. I will merge the description in with `createDonation`'s to further clarify.

Reply:
Done in 8525d41d53139507a331ed530ba06157bbcfdfc9